.travis/test.sh

```diff
@@ -1,13 +1,36 @@
-#!/bin/bash -e
+#!/bin/bash
+
+set -e
```

Comment:
Any reason to prefer this over `bash -e`?

Reply:
this also works if you execute `bash file` instead of `./file`